vitest.config.ts

```diff
@@ -1,35 +1,30 @@
-import path from 'node:path';
-import { fileURLToPath } from 'node:url';
-
-import { storybookTest } from '@storybook/addon-vitest/vitest-plugin';
-import react from '@vitejs/plugin-react';
+import react from '@vitejs/plugin-react-swc';
 import { defineConfig } from 'vitest/config';
 
-const dirname =
-  typeof __dirname !== 'undefined' ? __dirname : path.dirname(fileURLToPath(import.meta.url));
-
 export default defineConfig({
-  plugins: [react()],
+  plugins: [react() as any],
```

Comment:
Using `react() as any` to silence typing is a brittle workaround and defeats type-safety. The plugin import was changed to `@vitejs/plugin-react-swc` and cast to `any`.

**Sugerencia:**
```typescript
Prefer to keep proper plugin typing. If you must cast, add a comment explaining why and migrate to a typed import later. Example:

import type { Plugin } from 'vite';
const reactPlugin = react() as unknown as Plugin;
plugins: [reactPlugin],

But best is to import a plugin with correct type or use `@vitejs/plugin-react` if SWC plugin is not required.
```